content/2017-munich/sponsor.md

```diff
@@ -221,7 +306,7 @@ Thanks for your interest in sponsoring PromCon!
   loss to sponsors' properties by fire, theft, accident or any other cause
   whether the result of negligence or otherwise or case of force majeure.
 * Refund Policy - Please contact
-  [billing@robustperception.io](mailto:billing@robustperception.io) regarding
+  [Julius Volz](mailto:julius.volz@gmail.com) regarding
```

Comment:
Done. Before going live, we should test whether external people can send mails to that list.